lnbits/core/views/api.py

```diff
@@ -22,21 +22,30 @@
     WebSocketDisconnect,
 )
 from fastapi.exceptions import HTTPException
+from fastapi.params import Body
```

Comment:
this should be importet from `from fastapi import Body` doesnt it throw an mypy error?